core/jvmMain/src/kotlinx/serialization/SerializersJvm.kt

```diff
@@ -7,6 +7,7 @@
 
 package kotlinx.serialization
 
+import kotlinx.serialization.builtins.MapEntrySerializer
```

Comment:
unnecessary?

Reply:
Nope, triggers ambiguity between factory and internal class constructor